games/p.yaml

```diff
@@ -216,7 +216,8 @@
 
 - name: Progress Quest
   meta:
-    genre: [simulation]
+    genre: [MMORPG]
```

Comment:
This game is a parody of MMORPGs but it is a single-player game (the exact opposite of a MMO).